src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/BasicAccessioningService.java

```diff
@@ -147,9 +148,28 @@ public BasicAccessioningService(AccessionGenerator<MODEL, ACCESSION> accessionGe
     }
 
     @Override
-    public List<AccessionWrapper<MODEL, HASH, ACCESSION>> getByAccessions(List<ACCESSION> accessions,
-                                                                          boolean hideDeprecated) {
-        return dbService.findAllAccessionMappingsByAccessions(accessions);
+    public List<AccessionWrapper<MODEL, HASH, ACCESSION>> getByAccessionIds(List<ACCESSION> accessions,
```

Comment:
I'm not sure I understand the value of renaming "accessions" to "accessionIds", especially now that the same accession is repeated in several versions

Reply:
I'm not completely sure either, currently there are mentions to accessions as in the full object or accession mappings or wrappers. We should probably decide a terminology and enforce it.